telemetry/aiwrapper.go

```diff
@@ -0,0 +1,69 @@
+package telemetry
+
+import (
+	"github.com/Azure/azure-container-networking/aitelemetry"
+	"github.com/Azure/azure-container-networking/log"
+)
+
+var (
+	aiMetadata     string
+	th             aitelemetry.TelemetryHandle
+	gDisableTrace  bool
```

Comment:
what's g style? :)

Reply:
ya replied for jaeryn comment. g for global . im using disableTrace as functional parameter. Inorder to prevent conflict i added g at beginning. let me know if there is better way